homeassistant/components/insteon/__init__.py

```diff
@@ -149,17 +158,25 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             hass.config_entries.async_forward_entry_setup(entry, platform)
         )
 
+    device_registry = await hass.helpers.device_registry.async_get_registry()
     for address in devices:
         device = devices[address]
         platforms = get_device_platforms(device)
         if ON_OFF_EVENTS in platforms:
             add_on_off_event_device(hass, device)
+            device_registry.async_get_or_create(
+                config_entry_id=entry.entry_id,
+                identifiers={(DOMAIN, str(device.address))},
+                manufacturer="Smart Home",
```

Comment:
Can we add "Inc" to indicate it's a company.
```suggestion
                manufacturer="Smart Home, Inc.",
```

Reply:
A non-existent company now but yes, sure. :)

Reply:
I saw that I could make this a bit more DRY anyway.